libs/standalone-printf.c

```diff
@@ -0,0 +1,219 @@
+/* gcc -m32 -Wno-builtin-declaration-mismatch printf.c -o printf */
+/* clang -m32 -Wno-incompatible-library-redeclaration printf.c -Wno-implicit-function-declaration -o printf */
```

Comment:
Where does this come from?   I would think it would be a good idea to copy these files directly from bionic, but I guess for the tech demo its not super important to be 100% bug compatible with bionic?

Reply:
Yes, this is just a stopgap measure until we have the bionic version in place.
(This standalone mini-printf was something Nathan had lying around, and I beefed it up a tiny bit to make it work for us.)

Reply:
Dimitry is working on getting the bionic version in place, but I told him to make that second priority and focus on getting the toolchain solid.